ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2014, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+# prep the config for the running environment for zencommands (zodb)
+cp  -p /opt/zenoss/etc/zodb_db_main.conf /tmp/zodb_db_main.conf.sav
+cp  -p /opt/zenoss/etc/zodb_db_imp4.conf /opt/zenoss/etc/zodb_db_main.conf
```

Comment:
The original one is only needed before a commit.
In most operations, we return non-zero code to prevent unnecessary commits.